### PR TITLE
rebase off v2.6.1

### PR DESCRIPTION
**Pull request by kares**
\+ backported https://github.com/cloudvox/adhearsion/commit/e077bc6012015791ba81c87506b7f85af9da7ecf from upstream (master)

if travis-ci should be updated to point to PB branch (after https://github.com/cloudvox/punchblock/pull/22)

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,3 +1,46 @@
...
+  else
+    # NOTE: keep testing against AS 3.2.x by default
+    # :require needed as otherwise some specs fail :
+    # undefined method `deep_merge' for {:from=>"foo", :timeout=>3000, :headers=>{:x_foo=>"bar"}}:Hash
+    gem 'activesupport', '~> 3.2', require: ['active_support', 'active_support/core_ext/hash']
+  end
+  gem 'mime-types', '< 2.99'
+end
+
+if pb_version = ENV['PB_VERSION']
```
Comment (sfgeorge) [inline Gemfile:24]: Nice!  So I can just customize this environment variable in Travis to whatever we want, right?

**Comment by sfgeorge** [comment]
> \+ backported https://github.com/cloudvox/adhearsion/commit/e077bc6012015791ba81c87506b7f85af9da7ecf from upstream (master)

Excellent find!  Thanks for plucking this!

> if travis-ci should be updated to point to PB branch (after https://github.com/cloudvox/punchblock/pull/22)

File: Gemfile (hunk 1 above)
Comment (kares) [inline Gemfile:24]: yy that was the intention ... update [PB_VERSION](https://github.com/kares/adhearsion/blob/4927ad41eacd4199259c03da1ca305a76540f75d/.travis.yml#L23-L32) to `dialogtech/support/2.7.5` once merged

**Comment by sfgeorge** [comment]
Fantastic job, thank you very much Karol :+1: 

@ajayanty Can you merge this for us?

**Pull request merged**